Imabulary/Imabulary
# [Feature] [FE] - Add button to start the audio (IMB-69)

## Overview
This PR will add a word voiceover button on the flashcard screen

## Test cases

1. Open newly generated flashcard
2. Press on the volume_up button near the word
3. You should hear the voiceover of the word

## Issue

closes [IMB-69](https://www.notion.so/FE-Add-button-to-start-the-audio-d9de653468b447aa8cf3f387727b9941?pvs=4)
